server/plugin.go

```diff
@@ -182,9 +184,6 @@ func (p *Plugin) GetClientForUser(userID string) (msteams.Client, error) {
 	if token.Expiry.Before(time.Now()) {
 		newToken, err := client.RefreshToken(token)
 		if err != nil {
-			if msteams.IsOAuthError(err) {
-				p.OnDisconnectedTokenHandler(userID)
-			}
```

Comment:
Is this related to the metrics changes, or a fix for something else?

Reply:
No, it fix the duplication of the message because after the `make generate` it adds the `Refresh` to the disconnect layer and this special case is no longer needed.